src/main/java/ru/nmedvedev/rest/SodexoCustomClient.java

```diff
@@ -0,0 +1,36 @@
+package ru.nmedvedev.rest;
+
+import io.smallrye.mutiny.Uni;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+import ru.nmedvedev.model.Balance;
+import ru.nmedvedev.model.SodexoData;
+import ru.nmedvedev.model.SodexoResponse;
+
+import javax.enterprise.context.ApplicationScoped;
+
+@ApplicationScoped
+public class SodexoCustomClient {
```

Comment:
Seems like this class has only `getAmount` additional method. Couldn't it be moved to `SodexoClient` and be implemented as `default`?

Reply:
That was my first attempt but it failed. Don't know why - there're simply no request. Can't find any examples with microprofile and default method on the net

Reply:
Alright, let it be so